src/main/python/RemoteSwingLibrary.py

```diff
@@ -298,10 +302,14 @@ def set_java_tool_options(self):
         logger.debug("Set JAVA_TOOL_OPTIONS='%s'" % self._agent_command)
 
         t = tempfile.NamedTemporaryFile(prefix='grant_all_', suffix='.policy', delete=False)
-        t.write("""
+        text = """
             grant {
                 permission java.security.AllPermission;
-            };""")
+            };
+            """
+        if sys.version_info[0] >= 3:
+            text = bytes(text, "UTF-8")
```

Comment:
Could the above be just `b"""grant ..."""`? The `b` prefix makes it bytes in Python 3 and is a no-op on Python 2.

As a general note, could the code use `with tempfile.NamedTemporaryFile(...) as t` approach?